nkms/keystore/keypairs.py

```diff
@@ -76,7 +76,7 @@ def decrypt(self, message_kit: MessageKit) -> bytes:
         cleartext = pre.decrypt(ciphertext=message_kit.ciphertext,
                                 capsule=message_kit.capsule,
                                 priv_key=self._privkey,
-                                alice_pub_key=message_kit.alice_pubkey)
+                                alice_pub_key=message_kit.policy_pubkey)
```

Comment:
Ah yes, `policy_pubkey` is the former `alice_pubkey`. I don't like that name. What's wrong with `alice_pubkey`?

Reply:
Well, is it really alice's pubkey if she disappears from the internet, memory of her lost to the sands of time?  Does anybody at that point care who Alice is or was?  Isn't the only important thing that the key is the right key for that Policy?